src/templates/ApiEndpoint.tsx

```diff
@@ -612,98 +615,130 @@ export default function ApiEndpoint({ data, pageContext: { menu, breadcrumb, bre
     const overviewNode = allMdx.nodes?.find((node) => node.slug === `docs/api/${name}/overview`)
 
     return (
-        <ReaderView>
-            <SEO title={`${title} API Reference - PostHog`} />
-
-            <h2 className="!mt-0">{title}</h2>
-            <blockquote className="p-6 mb-4 rounded bg-accent">
-                <p>
-                    For instructions on how to authenticate to use this endpoint, see{' '}
-                    <a href="/docs/api/overview">API overview</a>.
-                </p>
-            </blockquote>
-
-            {overviewNode?.body && (
-                <div className="article-content mt-6">
-                    <MDXProvider components={components}>
-                        <MDXRenderer>{overviewNode.body}</MDXRenderer>
-                    </MDXProvider>
-                </div>
-            )}
+        <ScrollSpyProvider>
+            <ReaderView>
+                <div ref={contentContainerRef}>
+                    <SEO title={`${title} API Reference - PostHog`} />
+
+                    <h2 className="!mt-0">{title}</h2>
+                    <blockquote className="p-6 mb-4 rounded bg-accent">
+                        <p>
+                            For instructions on how to authenticate to use this endpoint, see{' '}
+                            <a href="/docs/api/overview">API overview</a>.
+                        </p>
+                    </blockquote>
+
+                    {overviewNode?.body && (
+                        <div className="article-content mt-6">
+                            <MDXProvider components={components}>
+                                <MDXRenderer>{overviewNode.body}</MDXRenderer>
+                            </MDXProvider>
+                            <SectionDivider />
+                        </div>
+                    )}
+
+                    <ReactMarkdown>{items[0].operationSpec?.description}</ReactMarkdown>
+
+                    <Endpoints paths={paths} containerRef={contentContainerRef} />
+
+                    {items.map((item, index) => {
+                        item = item.operationSpec
+                        const mdxNode = allMdx.nodes?.find((node) => node.slug.split('/').pop() === item.operationId)
+
+                        return (
+                            <div className="mt-8" key={item.operationId}>
+                                {index > 0 && <SectionDivider />}
+                                <div
+                                    className="grid grid-cols-1 lg:grid-cols-2 gap-8 items-start"
```

Comment:
Since everything has to be responsive within its container now, instead of...

before:

```
grid grid-cols-1 lg:grid-cols-2
```

after:
```
grid grid-cols-1 @xl:grid-cols-2
```

<img width="974" height="972" alt="image" src="https://github.com/user-attachments/assets/9b285939-273a-4892-83db-2457ce73dd2c" />